monai/inferers/utils.py

```diff
@@ -173,5 +174,5 @@ def _get_scan_interval(
             scan_interval.append(int(roi_size[i]))
         else:
             # scan interval is (1-overlap)*roi_size
-            scan_interval.append(int(roi_size[i] * (1 - overlap)))
+            scan_interval.append(math.ceil(int(roi_size[i]) * (1 - overlap)))
```

Comment:
perhaps 
```py
_interval = max(int(roi_size[i] * (1 - overlap)), 1)
scan_interval.append(_interval)
``` 
could preserve most of the previous behaviours?

Reply:
I feel that if we only use `ceil()` for `ROI=1` and still use `floor() or int()` for `ROI>1`, the logic will be different.
@ericspod , could you please help share some comments here?

Thanks.

Reply:
Using max as @wyli has it makes more sense I think, with ceil the size would be 1 larger than expected much of the time, `int(roi_size[i] * (1 - overlap))` will be 0 if `overlap` is near 1 itself in which case a interval of just 1 would work. 

This function in general though needs to be documented, the arguments explained, and the asserts replaced with checks.

Reply:
OK, updated in the latest commit.
Thanks.